src/structures/Manager.ts

```diff
@@ -64,124 +79,98 @@ export interface Manager {
     ): this;
 
     /**
-     * Emitted when track errors
-     * @event Manager#trackError
-     * @param {Player} player Player
-     * @param {TrackData} Track Error Track
-     */
-    on(
-        event: "trackError",
-        listener: (player: Player, track: TrackData) => void
-    ): this;
-
-    /**
-     * Emitted when Queue ends
+     * Emitted when the Queue ends.
      * @event Manager#queueEnd
      * @param {Player} player Player
      */
     on(event: "queueEnd", listener: (player: Player) => void): this;
 
     /**
-     * Emitted when Voice Connection is Ready
+     * Emitted when the Voice Connection is ready.
      * @event Manager#voiceReady
      * @param {Player} player Player
      */
     on(event: "voiceReady", listener: (player: Player) => void): this;
 
     /**
-     * Emitted when Voice Connection is disconnected
+     * Emitted when the Voice Connection is disconnected.
      * @event Manager#voiceDisconnect
      * @param {Player} player Player
      */
     on(event: "voiceDisconnect", listener: (player: Player) => void): this;
 
     /**
-     * Emitted when Voice Connection error
-     * @event Manager#voiceError
+     * Emitted when a new Player is created.
+     * @event Manager#playerCreate
      * @param {Player} player Player
-     * @param {Payload} payload raw payload from Nexus
      */
-    on(
-        event: "voiceError",
-        listener: (player: Player, payload: Payload) => void
-    ): this;
+    on(event: "playerCreate", listener: (player: Player) => void): this;
 
     /**
-     * Emitted when Audio Player Errors
-     * @event Manager#audioPlayerError
-     * @param {Player} player Player
-     * @param {Payload} payload raw payload from nexus
-     */
-    on(
-        event: "audioPlayerError",
-        listener: (player: Player, payload: Payload) => void
-    ): this;
-
-    /**
-     * Emitted when Player is created
-     * @event Manager#playerCreated
-     * @param {Player} player Player
-     */
-    on(event: "playerCreated", listener: (player: Player) => void): this;
-
-    /**
-     * Emitted when player is destroyed
+     * Emitted when a player is destroyed.
      * @event Manager#playerDestroy
      * @param {Player} player Old Player
      */
     on(event: "playerDestroy", listener: (player: Player) => void): this;
+
+    /**
+     * Emitted when there is an debuggable error caught.
+     * @event Manager#error
+     * @param {ChiroError} error The error containing details.
+     */
+    on(event: "error", listener: (error: ChiroEventError) => void): this;
 }
 
 /**
- * The Manager Class
+ * The Manager Class which manages all the players.
+ * 
  * @extends {EventEmitter}
  * @example
  * const manager = new Manager({
- *     node: {host: "localhost", port: 3000, password: "MySecurePassword"},
- *     send(id, payload){
+ *     node: { host: "localhost", port: 3000, password: "MySecurePassword" },
```

Comment:
```suggestion
 *     node: { host: "localhost", port: 3000, password: "SwagLordNitroUser12345" },
```

Password according to Nexus Docs. Should be used as default password

Reply:
It has already been fixed (https://github.com/DevSnowflake/chiro.js/pull/1/commits/c0735cfac723d81f078a11d779e50fea0c71fd2a).